From: Taylor R Campbell <campbell+mit-scheme@mumble.net>
Date: Thu, 4 Feb 2021 17:02:36 +0000
Subject: microcode: Use fallthrough attribute on clang 10 and 11 too.

(cherry picked from commit df76fc0ecbbc6bd0a0afc641abf655e86398d2b0)
---
 src/microcode/confshared.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/microcode/confshared.h b/src/microcode/confshared.h
index 502d1b1..f83671a 100644
--- a/src/microcode/confshared.h
+++ b/src/microcode/confshared.h
@@ -151,7 +151,7 @@ USA.
 #endif
 
 #if (((defined (__GNUC__)) && (__GNUC__ >= 7)) || \
-       ((defined (__clang__)) && (__clang_major__ >= 12)))
+       ((defined (__clang__)) && (__clang_major__ >= 10)))
 #  define FALLTHROUGH() ATTRIBUTE ((__fallthrough__))
 #else
 #  define FALLTHROUGH() ((void)0)
